lib/apple/index.js

```diff
@@ -112,8 +116,8 @@ exports.verifyPayment = function (payment, cb) {
 		return cb(null, result);
 	}
 
-
-	verify(apiUrls.production, { json: jsonData }, function (error, resultString) {
+	var url = sandbox ? apiUrls.sandbox : apiUrls.production;
```

Comment:
Why this need? We try production first, which will tell us with error code `21007` to hit the sandbox URL instead. That's already in place. Why the need to pass this sandbox boolean from the outside?